gradle/groovyProject.gradle

```diff
@@ -92,6 +92,8 @@ if (file("src/testFixtures").exists()) {
 
 apply from: "$rootDir/gradle/integTest.gradle"
 
+apply from: "$rootDir/gradle/crossVersionTest.gradle"
```

Comment:
This would require more changes in the whole setup. See: https://github.com/gradle/gradle/commit/1c25888c4996e29b965276f00356e6f16026c023